website/webpack.config.js

```diff
@@ -24,7 +24,6 @@ Encore
     })
 
     .disableSingleRuntimeChunk()
-    .cleanupOutputBeforeBuild()
```

Comment:
Moved this down to only run in prod. 

Couscous and webpack were both building assets on every change.